modules/ROOT/pages/shared-resources.adoc

```diff
@@ -4,7 +4,7 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: anypoint studio, shared resources, domains, multiple applications, share ports, domain project
 
-You can define connector configurations to be shared among all applications deployed under the same domain. To do so, create a Mule domain and then reference it from each application. As a result, each app now associated with the Mule domain can access shared resources in the domain. +
+When you deploy Mule on-premises, you can define global configurations such as default error handlers, shared properties, scheduler pools, and connector configurations to be shared among all applications deployed under the same domain. To do so, create a Mule domain and then reference it from each application. As a result, each app now associated with the Mule domain can access shared resources in the domain. +
```

Comment:
Remove the hyphen from "on-premises" because you are not using the phrase as an adjective here.